docs/competition/teams/role-descriptions/reception.md

```diff
@@ -30,27 +32,40 @@ If they still object, they should be given a red lanyard and told to wear them a
 
 ## Handling members of the public
 
-Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public is allowed into the building. You just need to inform them about photos and videos being recorded.
+Throughout the competition, receptionists typically get a lot of questions by the public. Any member of the public is allowed into the building.
 
 You can give them a booklet and direct them to the arena if they seem interested.
 
 ## Handling Team Arrivals
 
-When a team first arrives or leaves, your role at reception is to log this and confiscate items on entry.
+When a team first arrives or leaves, your role at reception is to log this, collect items from the team and inform them of the next steps.
+
+If competitors have arrived before their team supervisor, they should be directed to wait in the waiting area by reception until their team supervisor arrives.
 
 Offer each competitor a bag of swag as they enter.
 
 ### Team entry/exit recording
 
-You will have a sign-in/sign-out sheet for team supervisors to record that their teams have arrived, make sure they fill in the details and sign it.
+You will have been provided with access to the competition helpdesk system. One of the things this records is whether teams are present. When a team first arrives each day, please mark the team as arrived in the system. If a teams arrive and leave throughout the day, please update the system accordingly.
 
-### Confiscating batteries, chargers, and bags
+### Collecting batteries, chargers, and bags
 
 We handle battery charging and robot safety at the competition. This means we should take all of the following items from each team:
-    - 2 x Battery (Teams have 2 batteries, please take both).
-    - 1 x Battery charger.
-    - 1 x Battery charging bag.
-    - 1 x Battery charging Power Supply Unit (PSU).
+
+-   2 x Battery (Teams have 2 batteries, please take both)
+-   1 x Battery charger
+-   1 x Battery charging bag
+-   1 x Battery charging Power Supply Unit (PSU)
+
+### Next steps for the team
+
+Inform the team of these things when they first arrive at the venue:
+
+1. The team should proceed to their pit (Ushers and Shepherds will be available to provide directions).
+2. When ready for their first battery, the team should proceed to their nearest helpdesk.
+3. Helpdesk will perform a safety inspection of the robot and provide the team with a battery when it passes.
```

Comment:
Note: There aren't any documentation changes in this PR for helpdesk and what they need to do. Are they coming?